src/extensions.ts

```diff
@@ -0,0 +1,14 @@
+import Eris, { TextChannel } from "eris";
+
+Object.defineProperty(Eris.User.prototype, "tag", {
+  enumerable: false,
+  get: function () {
+    return `${this.username}#${this.discriminator}`;
+  },
+});
+Object.defineProperty(Eris.Message.prototype, "textChannel", {
+  enumerable: false,
+  get: function () {
+    return this.channel as TextChannel;
+  },
+});
```

Comment:
best not to clutter the index.ts with this stuff, and is pretty easily moved into it's on file